### PR TITLE
Feature/pda 48

### PR DESCRIPTION
**Pull request by gReis89**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: widgets/src/components/TaskList/TaskListContainer.js [hunk 1]
```diff
@@ -112,26 +127,54 @@ class TaskList extends React.Component {
...
+    const { connection } = this.state;
+
+    if (row.processId && row.id) {
+      const process = `${row.processId}@${row.id.split('@')[1]}`;
+      this.setState({ loading: true });
+      try {
+        const diagram = await getDiagram(connection, process);
+        this.setState({
+          diagramModal: {
+            title: `Process: ${process}`,
```
Comment (jeffgo10) [inline widgets/src/components/TaskList/TaskListContainer.js]: is this possible to use i18n in that title string there?

File: widgets/src/components/common/Table/custom/ActionCell.js [hunk 2]
```diff
@@ -5,21 +5,28 @@ import AssessmentIcon from '@material-ui/icons/Assessment';
...
-      text: 'Diagram',
-      icon: <AssessmentIcon fontSize="small" />,
-    },
-  ];
+    });
+  }
+
+  if (options.showClaim) {
+    menuItems.push({
+      text: 'Claim',
```
Comment (jeffgo10) [inline widgets/src/components/common/Table/custom/ActionCell.js]: is this possible to use i18n in that 'text' string here? if it is, pls do integrate and also some other parts in your branch. tnx

**Review by jeffgo10 — CHANGES_REQUESTED** [review]
only some i18n concerns. otherwise, looks good.

File: widgets/src/components/TaskList/TaskListContainer.js (hunk 1 above)
Comment (gReis89) [reply widgets/src/components/TaskList/TaskListContainer.js]: fixed, thanks for pointing that!

**Comment by entando-service** [comment]
All is well

**Review by jeffgo10 — DISMISSED** [review]
looks good! make sure you update the branch before merging

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by gReis89** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by werbth** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by werbth** [comment]
Retest this please

**Comment by entando-service** [comment]
All is well

File: widgets/src/api/pda/processes.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { METHODS, DOMAINS } from 'api/constants';
 
 import MOCK_PROCESSES from 'mocks/pda/processes';
+import MOCK_DIAGRAM from 'mocks/pda/diagram.svg';
 import makeRequest from 'api/makeRequest';
 
 // eslint-disable-next-line import/prefer-default-export
```
Comment (rimselis) [inline widgets/src/api/pda/processes.js]: not needed anymore

**Review by rimselis — DISMISSED** [review]

**Review by rimselis — APPROVED** [review]

**Comment by entando-service** [comment]
All is well

**Pull request merged**